daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1145,26 +1186,17 @@ private[lf] object SBuiltin {
             Control.Expression(e)
           }
 
-          machine.disclosureTable.contractById.get(SContractId(coid)) match {
```

Comment:
Priming on ledger contract cache with disclosed contracts now makes this code redundant